WooCommerce/Classes/ViewRelated/Search/SearchViewController.swift

```diff
@@ -128,12 +122,87 @@ class OrderSearchViewController: UIViewController {
 
         navigationController?.setNavigationBarHidden(false, animated: true)
     }
+
+    // MARK: - UITableViewDataSource Conformance
+    //
+
+    func numberOfSections(in tableView: UITableView) -> Int {
+        return resultsController.sections.count
+    }
+
+    func tableView(_ tableView: UITableView, numberOfRowsInSection section: Int) -> Int {
+        return resultsController.sections[section].numberOfObjects
+    }
+
+    func tableView(_ tableView: UITableView, cellForRowAt indexPath: IndexPath) -> UITableViewCell {
+        guard let cell = tableView.dequeueReusableCell(withIdentifier: Cell.reuseIdentifier, for: indexPath) as? Cell else {
+            fatalError()
+        }
+
+        let model = resultsController.object(at: indexPath)
+        let cellModel = searchCommand.createCellViewModel(model: model)
```

Comment:
Here is where I would explore getting rid of the cell view model, and pass model directly to the cell

Reply:
(as in https://github.com/woocommerce/woocommerce-ios/pull/1306#discussion_r328832085, `Order` search result cell requires more than just the model (`Order`) to configure the cell)